content/admin/org_settings/integrations/GenericWebhook.md

```diff
@@ -41,7 +41,19 @@ If you were using the Generic Webhook to create a VictorOps integration, the sam
 	"state_message":"$Message" 
 }
 ```
- 
+
+Similarly you can set up an integration with Microsoft Teams using the following payload:
```

Comment:
Simplify? Punctuation? Like:

You can integrate with Microsoft Teams in the same way. Use the following JSON payload: